core/src/main/scala/cats/syntax/reducible.scala

```diff
@@ -0,0 +1,19 @@
+package cats
+package syntax
+
+trait ReducibleSyntax1 {
+  implicit def foldableSyntaxU[FA](fa: FA)(implicit U: Unapply[Reducible,FA]): Reducible.Ops[U.M, U.A] =
+    new Reducible.Ops[U.M, U.A] {
+      val self = U.subst(fa)
+      val typeClassInstance = U.TC
+    }
+}
+
+trait ReducibleSyntax extends Reducible.ToReducibleOps with ReducibleSyntax1 {
+  implicit def nestedReducibleSyntax[F[_]: Reducible, G[_], A](fga: F[G[A]]): NestedReducibleOps[F, G, A] =
+    new NestedReducibleOps[F, G, A](fga)
+}
+
+class NestedReducibleOps[F[_], G[_], A](fga: F[G[A]])(implicit F: Reducible[F]) {
```

Comment:
Is this intentionally left open for extension? I tend to default to `final` on classes if I don't really expect people to extend them, but I may go overboard.